website/blog/2022-12-13-aluminium.md

```diff
@@ -0,0 +1,677 @@
+    <td>Commits since last release</td>
+    <td align="center">200</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">157</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">26</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center">76</td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">8</td>
+  </tr>
+</tbody>
+</table>
+
+For full details: [https://github.com/scalameta/metals/milestone/54?closed=1]
+(https://github.com/scalameta/metals/milestone/54?closed=1)
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs and
+Sublime Text. Metals is developed at the [Scala Center](https://scala.epfl.ch/)
+and [VirtusLab](https://virtuslab.com) with the help from
+[Lunatech](https://lunatech.com) along with contributors from the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- [Converting sbt style to mill style dependencies](#converting-sbt-style-to-mill-style-dependencies)
+- [Additional completions for dependencies](#additional-completions-for-dependencies)
+- [Actionable diagnotics](#actionable-diagnostic)
+- [Expose Test Cases as Separate Lenses](#expose-test-cases-as-separate-lenses)
+- [Add support for type definition](#add-support-for-type-definition)
+- [Allow rename for local names for non compiled code](#allow-rename-for-local-names-for-non-compiled-code)
+- [Enable run and debug Scala CLI scripts in Metals](#enable-run-and-debug-scala-cli-scripts-in-metals)
+- [Refactor package definition when moving files between packages](#refactor-package-definition-when-moving-files-between-packages)
+
+## Converting sbt style to mill style dependencies
+
+Thanks to [LaurenceWarne](https://github.com/LaurenceWarne) there is a new code
+actions that allows users to automatically convert sbt style defined
+dependencies to Mill ones whenever...
+
+## Additional completions for dependencies
+
+Add completions for `//> using lib @@`
+
+Adds completions for ivy"dependency@@" in build.sc file
+
+- Add sbt lib completions
+
+Ammonite Ivy completions for Scala 3
+
+feature: Add completions for plugins in ScalaCLI
+
+[jkciesluk](https://github.com/jkciesluk)
+
+## Actionable diagnostic
+
+[lwronski](https://github.com/lwronski)
+
+## Expose Test Cases as Separate Lenses
+
+[\#4569](https://github.com/scalameta/metals/pull/4569)
+([LaurenceWarne](https://github.com/LaurenceWarne))
+
+## Add support for type definition
+
+## Run main classes in shell when using code lenses
+
+More to come, tests not yet
+
+## Allow rename for local names for non compiled code
+
+## Enable run and debug Scala CLI scripts in Metals
+
+[\#4605](https://github.com/scalameta/metals/pull/4605)
+([lwronski](https://github.com/lwronski))
+
+## Refactor package definition when moving files between packages
+
+[\#4655](https://github.com/scalameta/metals/pull/4655)
+([susliko](https://github.com/susliko))
+
```

Comment:
Can you add something about the debugger?

```md
## Major upgrade of the debugger

Metals v0.11.10 ships a new major version of the Scala Debug Adapter, featuring better debugging steps (step filter), conditional breakpoints, logpoints and more. See a full list of features in the release notes of the Scala Debug Adapter [3.0.1](https://github.com/scalacenter/scala-debug-adapter/releases/tag/v3.0.1) and [3.0.3](https://github.com/scalacenter/scala-debug-adapter/releases/tag/v3.0.3).
```

Reply:
Yes! I was supposed to do that, but my memory failed me.